lib/cmd/common/build.js

```diff
@@ -190,6 +196,7 @@ function doBuild(args, cb) {
       if (keys.length > 0) {
         async.map(keys, common.waitFor, function(err, results) {
           if (err) {
+            console.log(err);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)